src/Tools/dotnet-monitor/Startup.cs

```diff
@@ -176,8 +176,8 @@ public void ConfigureServices(IServiceCollection services)
             app.UseAuthentication();
             app.UseAuthorization();
 
-            CorsConfiguration corsConfiguration = new CorsConfiguration();
-            Configuration.Bind(nameof(CorsConfiguration), corsConfiguration);
+            CorsConfigurationOptions corsConfiguration = new CorsConfigurationOptions();
+            Configuration.Bind(nameof(CorsConfigurationOptions), corsConfiguration);
```

Comment:
This changes the root option name from "CorsConfiguration" to "CorsConfigurationOptions". Please use `ConfigurationKeys.CorsConfiguration` to preserve the existing name in configuration.